doc/quick-start-user-guide.md

```diff
@@ -68,30 +60,65 @@ clientaddress = "192.168.56.101:24007"
 etcdcurls = "http://192.168.56.101:2379"
 etcdpurls = "http://192.168.56.101:2380"
 ```
-
 Replace the IP address accordingly on each node.
 
-**Start glusterd2 process:** Glusterd2 is not a daemon and currently can run only in the foreground.
+### Using external etcd instead of embedded etcd.
+
+By default glusterd2 uses embedded etcd, but glusterd2 also supports usage of external etcd.
+
+To provide external etcd details. Edit glusterd2 config file by adding `noembed` option and specifying
```

Comment:
A sentence cannot begin with `To provide external etcd details` and just end there ;)